src/content/it/fundamentals/performance/resource-prioritization.md

```diff
@@ -0,0 +1,369 @@
+voler aspettare che gli script siano caricati e pronti per elaborare il flusso.
+Preconnect ti aiuta a ridurre i tempi di attesa per un singolo roundtrip una
+volta che sei pronto per iniziare il recupero.
+
+## Prefetch
+
+`<link rel="prefetch">` è in qualche modo diverso da `<link rel="preload">` e
+`<link rel="preconnect">`, nel senso che non tenta di far succedere qualcosa di
+critico più velocemente; invece, cerca di far succedere qualcosa di non critico
+prima, se possibile.
+
+Lo fa informando il browser di una risorsa che dovrebbe essere necessaria come
+parte di una futura navigazione o interazione dell'utente, ad esempio,
+qualcosa che *potrebbe* essere necessario in seguito, se l'utente intraprende
+l'azione che siamo aspettando. Queste risorse vengono recuperate con la priorità
+**più bassa** in Chrome, quando viene caricata la pagina corrente e c'è
+larghezza di banda disponibile.
+
+Ciò significa che `prefetch` è più adatto a prevedere ciò che l'utente potrebbe
+fare in seguito e prepararsi, come recuperare la prima pagina dei dettagli del
+prodotto in un elenco di risultati o recuperare la pagina successiva in
+contenuto impaginato.
+
+    <link rel="prefetch" href="page-2.html">
+
+Tieni comunque presente che prefetch non funziona in modo ricorsivo.
+Nell'esempio precedente recupereresti l'HTML ma qualsiasi risorsa di cui 
+abbia bisogno `page-2.html` non viene scaricata prima del tempo a meno che tu
+non espliciti prefetch anche su di essa.
+
+### Prefetch non funziona come Override
+
+È importante notare che non è possibile utilizzare `<link rel="prefetch">` come
+metodo per ridurre la priorità di una risorsa esistente. Nel seguente HTML, si
+potrebbe pensare che dichiarare `optional.css` in un prefetch riduca la sua
+priorità del successivo `<link rel="stylesheet">`:
+
+    <html>
+      <head>
+        <link rel="prefetch" href="optional.css">
+        <link rel="stylesheet" href="optional.css">
+      </head>
+      <body>
+        Hello!
+      </body>
+    </html>
+
+Tuttavia, questo farà sì che il tuo foglio di stile venga prelevato due volte
+(anche con un potenziale cache hit la seconda volta), una volta con priorità
+predefinita **Massima** e una volta con priorità **Più bassa**, come prefetch
+per un recupero successivo:
+
+<figure>
+  <div class="aspect-ratio"
+       style="width: 1374px; --aspect-ratio-w: 1374; --aspect-ratio-h: 190">
+    <img src="images/res-prio-prefetch.png"
+         alt="Uno screenshot di Chrome Developer Tools che mostra optional.css che
+              viene recuperato due volte">
+  </div>
+</figure>
+
+Il doppio recupero può essere negativo per gli utenti. In questo caso, non solo
+avrebbero dovuto attendere il CSS che bloccava il rendering, ma avrebbero anche
+potenzialmente perso larghezza di banda scaricando il file due volte. Ricorda
+che la loro larghezza di banda potrebbe essere contabilizzata. Assicurati di
```

Comment:
che la loro larghezza di banda potrebbe essere in base al consumo. Assicurati di